templates/admin/citizen_project/members.html.twig

```diff
@@ -65,10 +65,6 @@
                                     {% endif %}
                                 </td>
                                 <td>
-                                    <a href="{{ path('app_admin_citizenproject_change_privilege', {citizenProject: citizen_project.id, adherent: administrator.id, privilege: constant('CITIZEN_PROJECT_FOLLOWER', membership), token: token}) }}"
-                                       class="btn btn-default btn-sm">
-                                        Transformer en follower
-                                    </a>
                                 </td>
```

Comment:
Should we leave this `<td>` empty ?

Reply:
need transform the entire table (all rows) on table of 2 cols and not 3 cols